examples/README.md

```diff
@@ -20,3 +20,37 @@ Additional examples for [Confluent Cloud](https://www.confluent.io/confluent-clo
 
 * [confluent_cloud.py](confluent_cloud.py): produces messages to Confluent Cloud and then reads them back again
 * [confluentinc/examples](https://github.com/confluentinc/examples/tree/master/clients/cloud/python): integrates Confluent Cloud and Confluent Cloud Schema Registry
+
+## venv setup
+
+It's usually a good idea to install Python dependencies in a virtual environment to avoid
+conflicts between projects.
+
+To setup a venv with the latest release version of confluent-kafka and dependencies of all examples installed:
+
+```
+> python3 -m venv venv_examples
+> source venv_examples/bin/activate
+> cd examples
+> pip install -r requirements.txt
+```
+
+To setup a venv that uses the current source tree version of confluent_kafka, you
+need to have a C compiler and librdkafka installed
+([from a package](https://github.com/edenhill/librdkafka#installing-prebuilt-packages), or
+[from source](https://github.com/edenhill/librdkafka#build-from-source)). Then:
+
+```
+> python3 -m venv venv_examples
+> source venv_examples/bin/activate
+> python setup.py build
```

Comment:
For this case it is better to do `python setup.py develop` instead of build and install.

Reply:
thanks